INSTALL

```diff
@@ -1,6 +1,26 @@
-# This text file should explain how to deploy your website.
 
-# You should write detailed description on:
+# Locally

```

Comment:
Can we change this to Local Development (or sth that involves the word "development")